packages/core/src/Stack/examples/VerticalStackExample.tsx

```diff
@@ -1,28 +1,40 @@
 import React from 'react';
-import { Stack, Flex } from '@contentful/f36-components';
+import { Stack, Box } from '@contentful/f36-components';
 import tokens from '@contentful/f36-tokens';
 
 export default function VerticalStackExample() {
-  const ColorBox = ({ backgroundColor, children }) => (
```

Comment:
I think having a component here is actually good and it allows to understand `<Stack>` code easier without stealing too much attention to children.

Reply:
Well, I don't really agree. The only code to understand here is `<Stack flexDirection="column" spacing="spacingS">`. In other examples, for Flex or Grid, we use Boxes directly. I think it is actually not needed to have this abstraction here and it makes it more consistent with other docs.